social_media/accounts/serializer.py

```diff
@@ -0,0 +1,46 @@
+from rest_framework import serializers
+
+from .models import User
+from datetime import datetime
+
+from upload.models import Posts
+from upload.serializer import PostSerializer
+
+
+class AccountRegistrationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['username', 'email', 'bio', 'password']
+
+    def create(self, validated_data):
+        user = User(
+            username=validated_data.get('username'),
+            email=validated_data.get('email')
+        )
+        user.created_on=datetime.now()
+        user.update_on=datetime.now()
+        user.set_password(validated_data.get('password'))
+        user.save()
+        return user
+
+    def update(self, instance, validated_data):
+        instance.username = validated_data.get('username')
+        instance.email = validated_data.get('email')
```

Comment:
Email should not be allowed to be updated like this. Products usually ask you to verify the email before updating it. Please remove support for email updation here